Add "Custom crash reporting" section to "Overview of APIs" (#2594)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/data-privacy.md

ms.topic: conceptual
ms.prod: microsoft-edge
ms.technology: webview
ms.date: 05/31/2023
---
# Data and privacy in WebView2

> The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.

Reviewer: Can we add a link to the Terms and Conditions?

Author: need URL for "WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License."

https://www.bing.com/search?q=WebView2+Runtime+Terms+and+Conditions

I'll check NuGet package site, and in VS, NuGet Package Manager.

Author: Likely: 
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30/License
but that's a version-specific URL

Author: closing in favor of new thread https://github.com/MicrosoftDocs/edge-developer/pull/2594/files#r1214885245

> The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.  To view the license, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section) and then click any of the download buttons, such as **Get the Link**, **Download**, or **x64**.  The license is displayed in a dialog.

Reviewer: ```suggestion
To view the license go to [Download the WebView2 Runtime](
https://developer.microsoft.com/microsoft-edge/webview2/#download-section)
where clicking on any of the download buttons, such as **Get the Link**, **Download**, or **x64**, will show the license in a dialog. 

```

> The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.  To view the WebView2 Runtime Terms and Conditions License, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), and then click a button, such as **Get the Link**, **Download**, or **x64**.  A dialog contains the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime.

Author: @maurawinstanley - which title string to use?  Currently this passage uses two different title strings:

"the WebView2 Runtime Terms and Conditions License"
"the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime"

> The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.  To view the WebView2 Runtime Terms and Conditions License, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), and then click a button, such as **Get the Link**, **Download**, or **x64**.  A dialog contains the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime.

Author: ```suggestion
The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the Microsoft Software License Terms for the Microsoft Edge WebView2 Runtime.  To view those License Terms, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), and then click a button, such as **Get the Link**, **Download**, or **x64**.  A dialog opens, containing the Microsoft Software License Terms for the Microsoft Edge WebView2 Runtime.
```

Author: suggested wording from NB: 
You can see the license in a dialog when click on any of the buttons to download.

> The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.  To view the WebView2 Runtime Terms and Conditions License, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), and then click a button, such as **Get the Link**, **Download**, or **x64**.  A dialog contains the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime.

Author: @maurawinstanley - see my suggested wording in [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2594/files#r1215039244)

> The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.  To view the WebView2 Runtime Terms and Conditions License, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), and then click a button, such as **Get the Link**, **Download**, or **x64**.  A dialog contains the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime.

Author: streamlined wording

ms.topic: conceptual
ms.prod: microsoft-edge
ms.technology: webview
ms.date: 06/02/2023
---
# Data and privacy in WebView2

The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.

> To view the WebView2 Runtime Terms and Conditions License, go to the [Microsoft.Web.WebView2](https://www.nuget.org/packages/Microsoft.Web.WebView2) NuGet package page, and then click the **License Info** link.  For example, [WebView2/1.0.1774.30/License](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30/License).

Author: @nishitha-burman @maurawinstanley 
Is this link & paragraph what we mean by "the WebView2 Runtime Terms and Conditions License"?
Is the name "the WebView2 Runtime Terms and Conditions License" correct?

https://www.nuget.org/packages/Microsoft.Web.WebView2
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30/License

Author: An open Issue from 2021 is about a different License that does say "terms" and "WebView2":  at page https://developer.microsoft.com/en-us/microsoft-edge/webview2/ 
in the "Evergreen Runtime Installer" section, click the "x64" button.  Opens a dialog in the browser:  

![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/45198fce-0823-414f-a2f1-a729170da83a)

Author: outdated, see comment https://github.com/MicrosoftDocs/edge-developer/pull/2594/files#r1215028818
